### DISCUSSION TITLE
Addresses in the UK

### DISCUSSION THREAD

**Discussion by tommycrock** [osmandapp/OsmAnd#13512] [Ideas]
The developers may be interested to know that there is an ongoing discussion about how to map addresses in the United Kingdom. This is in preparation for a project that will attempt to greatly increase the quantity and quality of addressed objects in the UK.

Current widely used OSM tags for addresses don't always work well in the UK as we have a sizeable minority of addresses where the 'house' numbers refer to a small location or sidestreet, and then a larger road is referenced. For example:
4 Faraday,
Netherfield Road
...
where addr:housenumber=4.

The main options currently being considered are:

A new address tag 'parentstreet' for the larger road, e.g.
addr:place=Faraday
addr:parentstreet=Netherfield Road
with this option, addr:street or addr:place would be used for the numbered thing depending on whether it was a highway or not.

A second new address tag 'substreet' for the smaller, numbered thing. This would be used in conjunction with parentstreet, e.g.
addr:substreet=Faraday
addr:parentstreet=Netherfield Road
substreet would be used whatever the numbered thing was. For more typical addresses addr:street would be used as normal.

It would be great if osmand supported whatever was settled on. For more information and to contribute please see the talk-gb list and archive (December '21 / Jan '22 "OSM UK address project: tags")
https://lists.openstreetmap.org/listinfo/talk-gb

**Comment by vshcherb** [comment]
That's quite interesting discussion though is not clear what I could add there... I think it will be always take at from example further and compare how it should work. As I see backward compatibility in simple scenarios is already taken into account
